docs/reference/errors-and-warnings/NU5501.md

```diff
@@ -0,0 +1,36 @@
+---
+title: NuGet Warning NU5501
+description: NU5501 Warning code
+author: zkat
+ms.author: kmarchan
+ms.date: 10/6/2020
+ms.topic: reference
+ms.reviewer: TBD
+f1_keywords: 
+  - NU5501
+---
+
+# NuGet Warning NU5501
+
+### Issue
```

Comment:
we should have a sample of the error message that gets displayed, so that customers who copy/paste the entire error message into a web search have a reasonable chance that the search engine will return this page.

It's a bit hard while the PR implementing the warning hasn't yet been merged, so I think this PR should be blocked until then. But we should have samples of all the different types of errors that can occur.

Reply:
Updated this.